### PR TITLE
Toggle k-th bit of a number in C++

### PR DESCRIPTION
**Pull request by zhcet19**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Resolved issue #4303

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Review by ankitaggarwal23 — COMMENTED** [review]
I'm not getting your problem exactly. 
- How many no. of times you're applying toggle to each bit?

It's better to write code for kth no. times. It's upto user how many times he/she many want to toggle bits.

**Comment by zhcet19** [comment]
@ankitaggarwal23 actually I was toggling all bits of a number only once but now i will change it to generalise it for k times.

**Review by ankitaggarwal23 — COMMENTED** [review]
Are you still working on it.

**Comment by zhcet19** [comment]
Yes, actually I was busy in semester exam, I will be working it by tomorrow

**Review by ankitaggarwal23 — COMMENTED** [review]
> Yes, actually I was busy in semester exam, I will be working it by tomorrow

All the best for your exams.

File: C-Plus-Plus/Bit--Manipulation/togglebit.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*
+ Progarm to Toggle kth bit of a number.
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/Bit--Manipulation/togglebit.cpp]: Fix typo its `program`

File: C-Plus-Plus/Bit--Manipulation/togglebit.cpp [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  
+  Ex:-6 (110) and k=2
+     after toggling  2nd bit our number would be (100) 4.
+ Approach:
+  firstly we will left shift the given number k-1 times.
+  After then we will xor the left shifted result with given number and return
+ */
+#include <bits/stdc++.h>
+using namespace std;
+int  toggle_k_bit(int n,int k)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/Bit--Manipulation/togglebit.cpp:14]: Add a new line after `std;`

File: C-Plus-Plus/README.md [hunk 3]
```diff
@@ -423,6 +423,7 @@ _add list here_
 - [Sort integers by the number of set bits present in them](Bit--Manipulation/sort_integers_by_their_number_of_set_bits.cpp)
 - [Swap_even_and_odd_bits](Bit--Manipulation/Swap_even_and_odd_bits.cpp)
 - [Number of Bits to be flipped to Inter-convert Two numbers](Bit--Manipulation/no_of_bits_to_be_flipped_to_inter_convert_two_numbers.cpp)
+- [Toggle each bit of a number](Bit--Manipulation/togglebit.cpp)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/README.md]: Rewrite it,
`[Toggle each bit of a number]` to `[Toggle k-th bit of a number]`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/README.md (hunk 3 above)
Comment (zhcet19) [reply C-Plus-Plus/README.md]: @ankitaggarwal23 i have made all the requested changes kindly review it

**Review by ankitaggarwal23 — APPROVED** [review]

**Review by ankitaggarwal23 — COMMENTED** [review]
Points updated

**Pull request merged**